src/s2wasm.h

```diff
@@ -1342,6 +1346,40 @@ class S2WasmBuilder {
     }
   }
 
+  // Converts invoke wrapper names generated by LLVM backend to real invoke
+  // wrapper names that are expected by JavaScript glue code.
+  // This is required to support wasm exception handling (asm.js style).
+  //
+  // LLVM backend lowers
+  //   invoke @func(arg1, arg2) to label %invoke.cont unwind label %lpad
+  // into
+  // ... (some code)
+  //   call @invoke_SIG(func, arg1, arg2)
+  // ... (some code)
+  // SIG is a mangled string generated based on the LLVM IR-level function
+  // signature. In LLVM IR, types are not lowered yet, so this mangling scheme
+  // simply takes LLVM's string representtion of parameter types and concatenate
+  // them with '_'. For example, the name of an invoke wrapper for function
+  // void foo(struct mystruct*, int) will be
+  // "__invoke_void_%struct.mystruct*_int".
+  // This function converts the names of invoke wrappers based on their lowered
+  // argument types and a return type. In the example above, the resulting new
+  // wrapper name becomes "invoke_vii".
+  template<typename ListType>
+  Name fixInvokeWrapper(const Name &name, WasmType result,
+                        const ListType &operands) {
+    return fixInvokeWrapper(name, getSig(result, operands));
+  }
+
+  Name fixInvokeWrapper(const Name &name, const std::string &sig) {
+    std::string name_str = name.c_str();
+    if (name_str.front() == '"' and name_str.back() == '"')
```

Comment:
Btw I learned about http://en.cppreference.com/w/cpp/keyword/and in exactly this same way... also TIL (from the appveyor build) that apparently MSVC doesn't support it :)